api/tests/integration/infrastructure/utils/pdf/certification-attestation-pdf_test.js

```diff
@@ -105,42 +105,58 @@ describe('Integration | Infrastructure | Utils | Pdf | Certification Attestation
   it('should generate a page per certificate', async function () {
     // given
     const resultCompetenceTree = domainBuilder.buildResultCompetenceTree();
-    const certificateWithCleaAndPixPlusDroitMaitre = domainBuilder.buildCertificationAttestation({
-      id: 1,
-      firstName: 'Jean',
-      lastName: 'Bon',
-      resultCompetenceTree,
-      certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_MAITRE_CERTIF }],
-    });
-    const certificateWithCleaAndPixPlusDroitExpert = domainBuilder.buildCertificationAttestation({
-      id: 2,
-      firstName: 'Harry',
-      lastName: 'Covert',
-      resultCompetenceTree,
-      certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_EXPERT_CERTIF }],
-    });
-    const certificateWithoutCleaNorPixPlusDroit = domainBuilder.buildCertificationAttestation({
-      ...certificateWithCleaAndPixPlusDroitMaitre,
-      id: 2,
-      firstName: 'Marc',
-      lastName: 'Decaffé',
-      cleaCertificationImagePath: null,
-      pixPlusDroitCertificationImagePath: null,
-      certifiedBadges: [],
-    });
+    const certificateWithCleaAndPixPlusDroitMaitreAndWithoutProfessionalizingMessage =
+      domainBuilder.buildCertificationAttestation({
+        id: 1,
+        firstName: 'Jean',
+        lastName: 'Bon',
+        resultCompetenceTree,
+        certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_MAITRE_CERTIF }],
+      });
+    const certificateWithCleaAndPixPlusDroitExpertAndWithProfessionalizingMessage =
+      domainBuilder.buildCertificationAttestation({
+        id: 2,
+        firstName: 'Harry',
+        lastName: 'Covert',
+        resultCompetenceTree,
+        certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_EXPERT_CERTIF }],
+        deliveredAt: new Date(),
+      });
```

Comment:
Expliciter le critère de date caché à now() > 01/01/2022 


``````suggestion
const deliveredOnStartDate = PROFESSIONALIZING_VALIDITY_START_DATE;
        
    const certificateWithCleaAndPixPlusDroitExpertAndWithProfessionalizingMessage =
      domainBuilder.buildCertificationAttestation({
        id: 2,
        firstName: 'Harry',
        lastName: 'Covert',
        resultCompetenceTree,
        certifiedBadges: [{ partnerKey: PIX_EMPLOI_CLEA_V3 }, { partnerKey: PIX_DROIT_EXPERT_CERTIF }],
        deliveredAt: deliveredOnStartDate,
      });
```

Reply:
Done.